gke-policies-v2/scalability/limits_pods_per_node.rego

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# METADATA
+# title: Number of PODs per node
+# description: The total number of PODs running on a single node
+# custom:
+#   group: Scalability
+#   severity: Low
+#   recommendation: >
+#     When the limit is reached, the kube-scheduler will assign the new PODs to the other nodes, if available.
+#     If there will be no nodes available to assign the POD, the cluster autoscaler, if configured,  may add new nodes.
```

Comment:
additional space between "if configured, " and "may add new nodes" ;-)

Reply:
👍